src/toil_lib/tools/aligners.py

```diff
@@ -186,3 +187,136 @@ def run_bwakit(job, config, sort=True, trim=False, mark_secondary=False):
     # Either write file to local output directory or upload to S3 cloud storage
     job.fileStore.logToMaster('Aligned sample: {}'.format(config.uuid))
     return job.fileStore.writeGlobalFile(os.path.join(work_dir, 'aligned.aln.bam'))
+
+
+def run_bowtie2(job,
+                read1,
+                name1, name2, name3, name4, rev1, rev2, ref,
+                read2=None):
```

Comment:
Is this being called directly from the tool that generates the index files? It might be a bit cleaner to pass a dict of IDs, but if you'd prefer it this way that's fine.